Extensions/better_reblogs.js

```diff
@@ -220,6 +220,16 @@ XKit.extensions.better_reblogs = new Object({
             if (!(this.preferences.margin.value || this.preferences.remove_user_names.value)) {
                 XKit.tools.add_css(".reblog-list-item .reblog-content {margin-left: 3px;}", "better_reblogs");
             }
+
```

Comment:
*be consistent WITHIN files
On Fri, Jan 8, 2016 at 10:12 PM Evan G <eg1290@gmail.com> wrote:

> Anyway, to kick this up a few meta levels, we should always be consistent
> be consistent between files, and changing indentation is out of scope for
> the PR
> On Fri, Jan 8, 2016 at 10:11 PM Evan G <eg1290@gmail.com> wrote:
>
>> Pretty sure the project is majority spaces, at least from the parts I've
>> worked with. Additionally, I've never seen the editorconfig file before.
>> (And have a very strong dislike of using tabs for code indentation)
>> On Fri, Jan 8, 2016 at 9:50 PM James Hobin <notifications@github.com>
>> wrote:
>>
>>> In Extensions/better_reblogs.js
>>> <https://github.com/new-xkit/XKit/pull/905#discussion_r49259914>:
>>>
>>> > @@ -220,6 +220,16 @@ XKit.extensions.better_reblogs = new Object({
>>> >              if (!(this.preferences.margin.value || this.preferences.remove_user_names.value)) {
>>> >                  XKit.tools.add_css(".reblog-list-item .reblog-content {margin-left: 3px;}", "better_reblogs");
>>> >              }
>>> > +
>>>
>>> The standard should be tabs. It's odd that this file doesn't conform to
>>> what is specified in the .editorconfig file
>>> <https://github.com/new-xkit/XKit/blob/master/.editorconfig>.
>>>
>>> —
>>> Reply to this email directly or view it on GitHub
>>> <https://github.com/new-xkit/XKit/pull/905/files#r49259914>.
>>>
>>